src/vivarium/framework/time.py

```diff
@@ -261,7 +261,7 @@ def setup(self, builder):
             if time.standard_step_size
             else self._minimum_step_size
         )
-        self._clock_step_size = self._standard_step_size
+        self._clock_step_size = self._minimum_step_size
```

Comment:
This seems like major change in logic - was this just a proper bug?

Reply:
In practice, the effect of this bit of code is just to determine what the duration of the very first timestep should be--_a priori_ either would work OK, but having the first step be a minimum step size is a bit more conservative and particularly better in the situation we see in MNCH where simulants have the minimum step size for the first 28 days of the simulation.